server/hwapi/endpoints/certification/logic.py

```diff
@@ -18,7 +18,7 @@
 #        Nadzeya Hutsko <nadzeya.hutsko@canonical.com>
 """The algorithms for determining certification status"""
 
-
+from typing import Sequence
```

Comment:
starting from Python 3.9 typing.T is deprecated for collections.abc.T https://stackoverflow.com/questions/75285711/difference-between-collections-abc-sequence-and-typing-sequence